CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.5.0] - UNRELEASED
+### Fixed
+- `import plotly.figure_factory` does not fail if `pandas` is not installed.
```

Comment:
Could you link to the PR where this was fixed?

Reply:
How's `This is fixed in the PR https://github.com/plotly/plotly.py/pull/958`?
or just `https://github.com/plotly/plotly.py/pull/958`